### PR TITLE
Manasa development

### PR DESCRIPTION
**Pull request by ashwatha**
Updated  the pages

### CODE REVIEW

File: app/views/households/create/householdcreateinfo.html [hunk 1]
```diff
@@ -19,18 +20,17 @@
       </div>
       <div class="col-sm-6 col-lg-12">
         <div class="form-group">
-          <label for="physicalAddressLine2" class="col-md-5 control-label"></label>
-          <div class="col-md-7">
-            <input class="form-control" name="householdAddressFormInput2" id="physicalAddressLine2" placeholder="" type="text" ng-model="req.household.physicalAddress.line2">
+          <label for="physicalAddressUnitNo" class="col-md-5 control-label">Unit No:</label>
```
Comment (justonian) [inline app/views/households/create/householdcreateinfo.html]: Can you call this physicalAddressUnit for consistency with the other changes? The label is fine, but the field name itself is what should match throughout the site.

File: app/views/households/create/householdcreateinfo.html [hunk 2]
```diff
@@ -19,18 +20,17 @@
...
+          <div class="col-md-9">
+            <input class="form-control" name="householdAddressFormInput2" id="physicalAddressUnitNo" placeholder="" type="text" ng-model="req.household.physicalAddress.unitNo">
```
Comment (justonian) [inline app/views/households/create/householdcreateinfo.html]: Same as above here.

File: app/views/households/create/householdcreateinfo.html [hunk 3]
```diff
@@ -155,7 +155,43 @@
...
         </div>
       </div>
-    </div>
+      <div class="col-sm-6 col-lg-12">
+        <div class="form-group">
+          <label for="ValidationType" class="col-md-5 control-label">Transportation:</label>
+          <div class="col-md-9">
+            <select class="form-control" name="transportation" id="transportation" ng-model="req.household.transportation">
+              <option></option>
+              <option value=""></option>
```
Comment (justonian) [inline app/views/households/create/householdcreateinfo.html]: Why are the option values blank here?

Comment (ashwatha) [inline app/views/households/create/householdcreateinfo.html]: I don't have the information on the HUD.

**Pull request merged**